suse_openstack_cloud

```diff
@@ -357,7 +357,7 @@ find_and_pconf_files /srv/www/openstack-dashboard/openstack_dashboard/local/loca
 #############################################################
 section_header "OpenStack log files"
 
-for svc in keystone rabbitmq glance swift nova cinder neutron heat ceilometer \
+for svc in keystone rabbitmq qpidd glance swift nova cinder neutron heat ceilometer \
```

Comment:
qpidd? really?

Reply:
well it was part of the sos-report plugins

Reply:
well it was part of the sos-report plugins

Reply:
@toabctl Are you querying this because we don't expect qpid to be part of the product?  I see https://build.suse.de/package/show/Devel:Cloud:8:Pike/qpid-proton and https://build.suse.de/package/show/Devel:Cloud:8:RHEL_Pike/qpid-proton but I don't know what it's needed for ...

Reply:
... but qpidd [is a thing](https://linux.die.net/man/1/qpidd), at least.

Reply:
> Are you querying this because we don't expect qpid to be part of the product? I see https://build.suse.de/package/show/Devel:Cloud:8:Pike/qpid-proton and https://build.suse.de/package/show/Devel:Cloud:8:RHEL_Pike/qpid-proton but I don't know what it's needed for ...

```qpid-proton``` is only needed as ```BuildRequires```.